documentation/book/con-tls-within-the-cluster.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+= TLS within the cluster
+
+All the internal communication within the Kafka cluster is encrypted with TLS. 
+
+== Zookeeper Communication
+
+Zookeeper does not support TLS itself. 
+By deploying an `stunnel` sidecar within every Zookeeper pod, the Cluster Operator is able to provide data encryption and authentication between Zookeeper nodes in a cluster.
+Zookeeper communicates only with the `stunnel` sidecar over the loopback interface.
+The `stunnel` sidecar then proxies all Zookeeper traffic, TLS decrypting data upon entry into a Zookeeper pod and TLS encrypting data upon departure from a Zookeeper pod.
+
+This TLS encrypting `stunnel` proxy is instantiated from the `spec.zookeeper.stunnelImage` specified in the Kafka resource.
+
+== Interbroker Kafka Communication
+
+Communication between Kafka brokers is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
+
+Communication between Kafka brokers and Zookeeper nodes uses an `stunnel` sidecar, as described above.
+
+== Topic and User Operators
+ 
+ The Topic and User Operators each use an `stunnel` sidecar, as described above when communicating with Zookeeper.
+ The Topic Operator connects to Kafka brokers on port 9091.
+
+== Kafka Client connections
+
+Encrypted communication between Kafka brokers and clients is provided through the `CLIENTTLS` listener on port 9093.
```

Comment:
Should we specify that it's true for clients running inside the same k8s/os cluster and then explaining what is used for clients outside? (different listener, port 9094 and so on))